addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -16,18 +16,11 @@ registry.category("web_tour.tours").add("ProductScreenTour", {
         [
             // Go by default to home category
             ProductScreen.clickShowProductsMobile(),
-            // Clicking product multiple times should increment quantity
-            ProductScreen.clickDisplayedProduct("Desk Organizer"),
-            ProductScreen.selectedOrderlineHas("Desk Organizer", "1.0", "5.10"),
-            ProductScreen.clickDisplayedProduct("Desk Organizer"),
-            ProductScreen.selectedOrderlineHas("Desk Organizer", "2.0", "10.20"),
 
-            // Clicking product should add new orderline and select the orderline
-            // If orderline exists, increment the quantity
-            ProductScreen.clickDisplayedProduct("Letter Tray"),
-            ProductScreen.selectedOrderlineHas("Letter Tray", "1.0", "5.28"),
             ProductScreen.clickDisplayedProduct("Desk Organizer"),
-            ProductScreen.selectedOrderlineHas("Desk Organizer", "3.0", "15.30"),
+            ProductScreen.clickDisplayedProduct("Desk Organizer"),
```

Comment:
check the price and quantity, also the previous call to `ProductScreen.clickDisplayedProduct`